### PR TITLE
docs(profile): Setting Profile Avatar

### LINKED ISSUE #117: Document an example how to set profile avatar
Documentation on how to set this needs to be added here: https://developers.verida.io/docs/client-sdk/profiles

### PR DESCRIPTION
**Pull request by cmcWebCode40**
- Added a sample code for setting profile Avatar
closes #117

### CODE REVIEW

**Review by dmty — APPROVED** [review]

File: docs/client-sdk/profiles.md [hunk 1]
```diff
@@ -59,16 +59,25 @@ const externalProfile = await context.openProfile('public', did);
 Open your own profile for the current context:
 
 ```jsx
+const avatarUri = 'data:image/png;base64,iVBOR...' // An  encoded as a URI
```
Comment (aurelticot) [inline docs/client-sdk/profiles.md:62]: Can you move this line just above the line `await myProfile.set('avatar', { uri: avatarUri });`?
Can you also add the missing `;` at the end of the line?
Can you modify the comment `// Data URL of base64-encoded image`

File: docs/client-sdk/profiles.md [hunk 2]
```diff
@@ -59,16 +59,25 @@ const externalProfile = await context.openProfile('public', did);
...
+
 const myProfile = await context.openProfile('public');
+
 await myProfile.set('name', 'Stevie');
+
+await myProfile.set('avatar', { uri: avatarUri, });
+
 const name = await myProfile.get('name', 'Stevie');
```
Comment (aurelticot) [inline docs/client-sdk/profiles.md:70]: Remove the wrong second parameter in the get method, just: `.get('name)`

File: docs/client-sdk/profiles.md [hunk 3]
```diff
@@ -59,16 +59,25 @@ const externalProfile = await context.openProfile('public', did);
 Open your own profile for the current context:
 
 ```jsx
+const avatarUri = 'data:image/png;base64,iVBOR...' // An  encoded as a URI
+
 const myProfile = await context.openProfile('public');
+
 await myProfile.set('name', 'Stevie');
```
Comment (aurelticot) [inline docs/client-sdk/profiles.md:66]: Add a line above with a comment
`// Modify the profile properties`

File: docs/client-sdk/profiles.md (hunk 2 above)
Comment (aurelticot) [inline docs/client-sdk/profiles.md:70]: Add a line above with a comment
`// Read my profile properties`

Comment (aurelticot) [inline docs/client-sdk/profiles.md:70]: Add a line below with
`console.log(name); // -> Stevie`

File: docs/client-sdk/profiles.md [hunk 4]
```diff
@@ -59,16 +59,25 @@ const externalProfile = await context.openProfile('public', did);
...
+
+await myProfile.set('avatar', { uri: avatarUri, });
+
 const name = await myProfile.get('name', 'Stevie');
 
 const profileData = await myProfile.getMany();
+
 await myProfile.delete('name');
```
Comment (aurelticot) [inline docs/client-sdk/profiles.md:74]: Add a line above with comment
`// Delete a profile property`

File: docs/client-sdk/profiles.md [hunk 5]
```diff
@@ -59,16 +59,25 @@ const externalProfile = await context.openProfile('public', did);
...
+
 const myProfile = await context.openProfile('public');
+
 await myProfile.set('name', 'Stevie');
+
+await myProfile.set('avatar', { uri: avatarUri, });
+
 const name = await myProfile.get('name', 'Stevie');
 
 const profileData = await myProfile.getMany();
```
Comment (aurelticot) [inline docs/client-sdk/profiles.md:72]: Add a line below with
`console.log(profileData); // -> { name: 'Stevie', avatar: { uri: 'data:image/png;base64,iVBOR...' }, ...}`

**Review by aurelticot — CHANGES_REQUESTED** [review]
- [x] Can you remove the useless title line 53 "Load Profile avatar ..."
- [x] Can you update the title line 57 from "Open my profile" to "Open and modify my profile"
- [x] Apply other inline comments
- [ ] Add a callout below the code block, similar to the info callout [here](https://developers.verida.io/docs/extensions/verifiable-credentials#issuing-a-credential) with something like: "Learn more about Data URL [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URLs)"

File: docs/client-sdk/profiles.md (hunk 1 above)
Comment (cmcWebCode40) [reply docs/client-sdk/profiles.md]: fixed

File: docs/client-sdk/profiles.md (hunk 3 above)
Comment (cmcWebCode40) [reply docs/client-sdk/profiles.md:66]: fixed

File: docs/client-sdk/profiles.md (hunk 2 above)
Comment (cmcWebCode40) [reply docs/client-sdk/profiles.md]: fixed

File: docs/client-sdk/profiles.md (hunk 5 above)
Comment (cmcWebCode40) [reply docs/client-sdk/profiles.md:71]: fixed

**Comment by cmcWebCode40** [comment]
@aurelticot , i have updated the this PR with the requested changes

File: docs/client-sdk/profiles.md (hunk 4 above)
Comment (aurelticot) [reply docs/client-sdk/profiles.md:75]: Can you add this comment line?

File: docs/client-sdk/profiles.md (hunk 2 above)
Comment (aurelticot) [reply docs/client-sdk/profiles.md]: Can you add this `console.log` line under the `get('name')`?
```JavaScript
const name = await myProfile.get('name');
console.log(name); // -> Stevie
```

File: docs/client-sdk/profiles.md [hunk 6]
```diff
@@ -50,28 +50,44 @@ const did = 'did:vda:0x6B2a1bE81ee770cbB4648801e343E135e8D2Aa6F';
...
+
+// Modify the profile properties
 await myProfile.set('name', 'Stevie');
-const name = await myProfile.get('name', 'Stevie');
+
+// Read my profile properties
+const name = await myProfile.get('name');
 
 const profileData = await myProfile.getMany();
+
```
Comment (aurelticot) [inline docs/client-sdk/profiles.md:72]: Can you delete this empty line 72?
```JavaScript
const profileData = await myProfile.getMany();
console.log(profileData); // -> ...
```

File: docs/client-sdk/profiles.md [hunk 7]
```diff
@@ -50,28 +50,44 @@ const did = 'did:vda:0x6B2a1bE81ee770cbB4648801e343E135e8D2Aa6F';
...
+## Open and modify my profile
 
 Open your own profile for the current context:
 
 ```jsx
+
 const myProfile = await context.openProfile('public');
+
+const avatarUri = 'data:image/png;base64,iVBOR...'; //  Data URL of base64-encoded image
+
```
Comment (aurelticot) [inline docs/client-sdk/profiles.md:62]: Can you remove this empty line 62 and arrange the whole like this?
```JavaScript
const myProfile = await context.openProfile('public');

// Modify the profile properties
await myProfile.set('name', 'Stevie');
const avatarUri = 'data:image/png;base64,iVBOR...'; //  Data URL of base64-encoded image
await myProfile.set('avatar', { uri: avatarUri, });

```

File: docs/client-sdk/profiles.md [hunk 8]
```diff
@@ -50,28 +50,44 @@ const did = 'did:vda:0x6B2a1bE81ee770cbB4648801e343E135e8D2Aa6F';
...
+await myProfile.set('avatar', { uri: avatarUri, });
+
+// Modify the profile properties
 await myProfile.set('name', 'Stevie');
-const name = await myProfile.get('name', 'Stevie');
+
+// Read my profile properties
```
Comment (aurelticot) [inline docs/client-sdk/profiles.md:68]: Can you change "my" by "the"
```JavaScript
// Read the profile properties
```

File: docs/client-sdk/profiles.md [hunk 9]
```diff
@@ -50,28 +50,44 @@ const did = 'did:vda:0x6B2a1bE81ee770cbB4648801e343E135e8D2Aa6F';
 const externalProfile = await context.openProfile('public', did);
 ```
 
-## Load profile avatar and name
-
-
-
-## Open my profile
+## Open and modify my profile
```
Comment (aurelticot) [inline docs/client-sdk/profiles.md:53]: Can you change this title as follow:
`## Open and modify the current user's profile`

File: docs/client-sdk/profiles.md [hunk 10]
```diff
@@ -50,28 +50,44 @@ const did = 'did:vda:0x6B2a1bE81ee770cbB4648801e343E135e8D2Aa6F';
...
 
 Open your own profile for the current context:
```
Comment (aurelticot) [inline docs/client-sdk/profiles.md:55]: Can you change this sentence by:
"Open the user's context profile"

File: docs/client-sdk/profiles.md [hunk 11]
```diff
@@ -50,28 +50,44 @@ const did = 'did:vda:0x6B2a1bE81ee770cbB4648801e343E135e8D2Aa6F';
...
 
 ```jsx
+
 const myProfile = await context.openProfile('public');
```
Comment (aurelticot) [inline docs/client-sdk/profiles.md:59]: Can you change the property name `myProfile` by `userProfile` everywhere in this sample code:
```JavaScript
const userProfile = await context.openProfile('public');

...
```

**Review by aurelticot — CHANGES_REQUESTED** [review]

File: docs/client-sdk/profiles.md (hunk 4 above)
Comment (cmcWebCode40) [reply docs/client-sdk/profiles.md]: resolved

File: docs/client-sdk/profiles.md (hunk 6 above)
Comment (cmcWebCode40) [reply docs/client-sdk/profiles.md:65]: modified

File: docs/client-sdk/profiles.md (hunk 7 above)
Comment (cmcWebCode40) [reply docs/client-sdk/profiles.md]: modified

File: docs/client-sdk/profiles.md (hunk 8 above)
Comment (cmcWebCode40) [reply docs/client-sdk/profiles.md]: modified

File: docs/client-sdk/profiles.md (hunk 10 above)
Comment (cmcWebCode40) [reply docs/client-sdk/profiles.md]: modified

**Review by aurelticot — APPROVED** [review]
Ok

**Pull request merged**